data/SpritePointersList.s

```diff
@@ -24,37 +25,37 @@ battleSpritePtrs:: .word battleSpriteMegaMan
 	.word battleSprite_823B768
 	.word spriteWhiteDot
 virusBattleSpritePtrs:: .word spriteWhiteDot
-	.word compVirusBattleSpriteMetteur + 1<<31
-	.word compVirusBattleSprite_8242E94 + 1<<31
+	.word compVirusBattleSpriteMetteur + SPRITE_IS_COMPRESSED
```

Comment:
You might want to replace the `SPRITE_IS_COMPRESSED` constant with `COMPRESSED_PTR_FLAG` constant.
```suggestion
	.word compVirusBattleSpriteMetteur + COMPRESSED_PTR_FLAG
```